app/kumactl/cmd/install/install_control_plane.go

```diff
@@ -102,14 +104,18 @@ var DefaultInstallControlPlaneArgs = InstallControlPlaneArgs{
 
 var InstallCpTemplateFilesFn = InstallCpTemplateFiles
 
+// InstallWithConnectionToKubernetes leverage connection to Kubernetes to preserve resources like autogenerated Secrets and not override them on every kumactl install
+var InstallWithConnectionToKubernetes = true
```

Comment:
This is more about the test now. Shall we consider putting this as command-line argument too, in case someone decides they want to enforce it?